client/components/QuestionsAnswers/QuestionsAnswers.jsx

```diff
@@ -1,13 +1,264 @@
 import React from 'react';
-import { COLORS } from '../../settings/colors';
 import styled from 'styled-components';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import axios from 'axios';
+import api from '../../api.js';
 
-const QuestionsAnswers = () => (
-  <Header>Q&A Module</Header>
-);
+
+// const AnswerTitle = styled.div`
+//   display: inline-block
+// `
+
+// const AnswerBody = styled.div`
+//   display: inline-block
+// `
+
+const Wrapper = styled.div`
+  display: flex
+
+`;
+
+const Photos = styled.div`
+  float: left
+`;
+
+const ByUser = styled.div`
+  clear: both
+`;
+
+const QuestionsHidden = styled.div`
+  display: none
+`;
+
+const AnswersHidden = styled.div`
+  display: none
+`;
+
+const AlignRight = styled.span`
+  float: right
+`;
+
+const YesLink = styled.u`
+
+`;
+
+
+
+class QuestionsAnswers extends React.Component {
+  constructor(props) {
+    super(props);
+    this.addHelpful = this.addHelpful.bind(this);
+
+    this.state = {
+      answers: [],
+      questions: []
+    };
+  }
+
+  componentDidMount() {
```

Comment:
We should chat this week - I think we might end up having to pull this logic up into <App> I'm not sure we want to be pinging the API every time the component mounts.